cassovary-core/src/main/scala/com/twitter/cassovary/algorithms/centrality/AbstractCentrality.scala

```diff
@@ -17,7 +17,7 @@ import collection.immutable
 import com.twitter.cassovary.graph.{Node, DirectedGraph}
 
 /**
- * Centrality algorithm abstraction that requires a directed graph {@code graph} be passed upon
+ * Centrality algorithm abstraction that requires a directed graph (<code>graph</code>) be passed upon
```

Comment:
Ha.  Yep.  Markdown definitely made it look exactly like < code > graph < /code >